app/controllers/layouts_controller.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class LayoutsController < ApplicationController
```

Comment:
I was think how to do that, and try to find the way, but I didn't, so I chose something that I know, also I had felling that there will be kind of this, but I could find it, probably because I was looking for kind of rendering but it was only by js :)